data/recommendation/recWindow.html

```diff
@@ -5,9 +5,10 @@
   </head>
   <body>
     <div id="welcome" class="welcome-box">
-      <h1>Change your password for this site immediately</h1>
-      <p>This site was recently hacked, and your account credientials
-      may have been compromised. If you use your current password for this site on
+      <h1 id="title">Change your password for this site immediately!</h1>
+      <p id="date">This site was compromised recently.</p>
+      <p id="count">It affected many users, and you may be among them.</p>
+      <p>If you use your current password for this site on
       any other sites, we recommend you change those as well.</p>
```

Comment:
Nit: If you have to break HTML tags across multiple lines, you should indent it to show the tree structure, like:

```html
<p id="count">It affected many users, and you may be among them.</p>
<p>
  If you use your current password for this site on
  any other sites, we recommend you change those as well.
</p>
```